.circleci/config.yml

```diff
@@ -138,15 +138,10 @@ workflows:
               only: main
       - create-xcframework
 
-  ios-release:
+  release:
     jobs:
-      - ios-build-release:
-          name: "iOS Release Build"
-          filters:
-            branches:
-              only: /Release\/.*/
-      - ios-release-tag:
-          name: "iOS Release Tag & Docs"
+      - release:
+          name: "Release"
           filters:
             tags:
               only: /.*/
```

Comment:
Should we run release job for v-tags only?

Reply:
Yes. I think there's a step in this workflow that checks for that (and skips the job if the tag doesn't match), but maybe we can refactor it to just do that check here instead.

Reply:
My point is that's it better to not run a workflow if it wouldn't work. The only change we have to make is:
```suggestion
              only: /v.*/
```

Reply:
Updated — I just transferred the existing regex we were using within the job